cli/command/container/stats.go

```diff
@@ -175,12 +175,12 @@ func runStats(dockerCli *command.DockerCli, opts *statsOptions) error {
 		for _, c := range cStats.cs {
 			cErr := c.GetError()
```

Comment:
No problem! Just started doubting myself if I overlooked something, and you may have had a special reason not to change it haha 👍